src/pages/[platform]/build-a-backend/auth/manage-users/manage-devices/index.mdx

```diff
@@ -326,6 +326,26 @@ Future<void> fetchAllDevices() async {
 ```
 
 </InlineFilter>
+
+<InlineFilter filters={["flutter"]}>
```

Comment:
while this should render okay when the platform is flutter, this is in the middle of a filter block for `fetchDevices` across the platforms in the mdx file. can we move this down past the block for the swift example on line 448? https://github.com/aws-amplify/docs/pull/7737/files#diff-d31670d1fd9a61a4eb6d4ff02d10fa76390de4ef93f6d7744047d4d92bf9238bR448